packages/verified-fetch/src/index.ts

```diff
@@ -750,6 +758,14 @@ export interface CreateVerifiedFetchOptions {
    * @default 60000
    */
   sessionTTLms?: number
+
+  /**
+   * Whether to include server-timing headers in responses. This option can be overridden on a per-request basis.
+   *
+   * @default false
+   * @see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Server-Timing
+   */
+  includeServerTiming?: boolean
```

Comment:
Since it's typed as a boolean, any thoughts on making this a bit shorter?
```suggestion
  serverTiming?: boolean
```

Reply:
I don't have a strong preference, but I prefer a more explicit name since `serverTiming` doesn't communicate its purpose. Also, verb-prefixed names for booleans are more idiomatic. What do you think about `withServerTiming` instead?

Option names should communicate their intent and effect rather than aim for brevity. More explicit names lead to a better dev understanding, with and without reading the docs. Also, the options are only passed once.

Reply:
Fair enough! I agree. 

`withServerTiming` works too and is shorter.